packages/native-stack/src/views/NativeStackView.tsx

```diff
@@ -73,7 +73,12 @@ export default function NativeStackView({
             <View
               style={[
                 styles.container,
-                { backgroundColor: colors.background },
+                {
+                  backgroundColor:
+                    stackPresentation !== 'transparentModal'
```

Comment:
How about the old configuration `transparentCard:true`. Does this settings works too

Reply:
This option is not related with the changed package (native-stack) and this option was [removed in v4](https://github.com/react-navigation/stack/blob/48b1651e99e3feb01c3caa5e0cab87b24b90ccf0/src/utils/validateDeprecatedConfig.tsx#L16).